ldms/src/sampler/lustre_client/lustre_client.c

```diff
@@ -23,8 +23,8 @@
 
 /* locations where llite stats might be found */
 static const char * const llite_paths[] = {
-        "/proc/fs/lustre/llite",          /* lustre pre-2.12 */
         "/sys/kernel/debug/lustre/llite"  /* lustre 2.12 and later */
+        "/proc/fs/lustre/llite",          /* lustre pre-2.12 */
```

Comment:
This is buggy. There is no comma between the two strings, so they are merged into one string by the compiler.